### PR TITLE
Add Node Matrix back into the ESLint checks

### PR DESCRIPTION
**Pull request by jessebeach**
Specify an explicit Node version for the ESLint check.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -50,6 +50,9 @@ jobs:
   eslint:
     name: ESLint (Node v${{ matrix.node }})
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [16]
```
Comment (nschonni) [inline .github/workflows/test.yml:55]: You need to use this in the setup-node section if you want to pin it. I removed that line when I unpinned from 12. By default it should use the current LTS, which is 16, already

Comment (jessebeach) [reply .github/workflows/test.yml:55]: @nschonni, Do you know how to interpolate the version that gets used into the output? Without the matrix config, the line `ESLint (Node v${{ matrix.node }})` returns null for the `${{ matrix.node }}` value. Maybe the `runner` has a param for the version of Node that gets loaded?

Comment (nschonni) [reply .github/workflows/test.yml:55]: Yeah, I should have cleaned up the title when I removed the matrix value in the setup-node `with` section.
I would probably suggest cleaning up the title unless you want to keep it pinned.

Comment (jessebeach) [reply .github/workflows/test.yml:55]: @nschonni I am OK with keeping it pinned. It'll be valid for another year plus :)

Comment (nschonni) [reply .github/workflows/test.yml:55]: OK, you'd just need to add back the line like on https://github.com/A11yance/aria-query/pull/312/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R23 to get it to pick it up

Comment (jessebeach) [reply .github/workflows/test.yml:55]: Good call. Fixed in #314

**Pull request merged**